biztalk/TOC.md

```diff
@@ -390,6 +391,7 @@
 ### [Upgrade](install-and-config-guides/upgrade-to-biztalk-server-2013-and-2013-r2.md)
 ## [Install a Multi-Computer scenario](install-and-config-guides/install-biztalk-server-in-a-multi-computer-environment.md)
 ## [Configure BizTalk](install-and-config-guides/configure-biztalk-server.md)
+### [Configure BizTalk](install-and-config-guides/gmsa-config.md)
```

Comment:
Text is same for sub-page. it can be "configure BizTalk gMSA"

Reply:
thanks! updated